<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-PIKCM99" ignored="0" name="Test" tests="4" failures="1" timestamp="2021-02-27T10:57:47 IST" time="49.204" errors="0">
  <testcase name="test" time="0.021" classname="testNgpackage.NewTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [hello] but found [Hello]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [hello] but found [Hello]
at testNgpackage.NewTest.test(NewTest.java:37)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.002" classname="testNgpackage.NewTest"/>
  <testcase name="test" time="0.006" classname="testNgpackage.NewTest"/>
  <testcase name="f" time="30.45" classname="testNgpackage.NewTest"/>
</testsuite> <!-- Test -->
